man/systemd.network.xml

```diff
@@ -1130,6 +1130,39 @@
         </varlistentry>
       </variablelist>
   </refsect1>
+  <refsect1>
+    <title>[BridgeVLAN] Section Options</title>
+      <para>The <literal>[BridgeVLAN]</literal> section manages the
+      VLAN ID configuration of a bridge port and accepts the following
+      keys. Specify several <literal>[BridgeVLAN]</literal> sections to
+      configure several VLAN entries. The VLANFiltering= option has to be
+      enabled.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>VLAN=</varname></term>
+          <listitem>
+            <para>The VLAN ID allowed on the port. This can be either a
+            single ID or a range M-N.</para>
```

Comment:
Please mention the the valid range for the IDs. i.e positive integer > 0, or so...